src/new/templates/rust/no-ui/hypr-echo/hypr-echo/src/lib.rs

```diff
@@ -0,0 +1,71 @@
+use serde::{Serialize, Deserialize};
+use hyperware_process_lib::{
+    LazyLoadBlob,
+    http::server::{WsMessageType, send_ws_push}
+};
+use hyperprocess_macro::hyperprocess;
+
+#[derive(Default, Debug, serde::Serialize, serde::Deserialize)]
+pub struct HyprEchoState {}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct HyprEchoReq {
+    payload: String,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct HyprEchoResp {
+    payload: String,
+}
+
+#[hyperprocess(
+    name = "HyprEcho",
+    ui = None,
+    endpoints = vec![
+        Binding::Http {
+            path: "/api",
+            config: HttpBindingConfig::new(false, false, false, None),
+        },
+        Binding::Ws {
+            path: "/ws",
+            config: WsBindingConfig::new(false, false, false),
+        }
+    ],
+    save_config = SaveOptions::Never, // Changed as state is removed
+    wit_world = "hypr-echo-template-dot-os-v0"
+)]
+
+
+impl HyprEchoState {
+    // Initialize the process, every application needs an init function
+    #[init]
+    async fn initialize(&mut self) {
+        println!("init HyprEcho");
+    }
+
+    // Endpoint accepting both local, remote Hyperware requests, and HTTP requests
+    #[local]
+    #[remote]
+    #[http]
+    async fn echo(&self, req: HyprEchoReq) -> HyprEchoResp {
+        if req.payload == "echo" {
+            HyprEchoResp { payload: "echo".to_string() }
+        } else {
+            HyprEchoResp { payload: "don't talk to me".to_string() }
+        }
+    }
```

Comment:
Probably the echo behaviors should match. Compare to the existing echo: https://github.com/hyperware-ai/kit/blob/master/src/new/templates/rust/no-ui/echo/echo/src/lib.rs